integrations/api_repo_test.go

```diff
@@ -405,6 +405,27 @@ func testAPIRepoMigrateConflict(t *testing.T, u *url.URL) {
 	})
 }
 
+// mirror-sync must fail with "400 (Bad Request)" when an attempt is made to
```

Comment:
note: I haven't contributed API tests so please guide me if the test looks awful.

Reply:
Tests seem reasonable, and very much appreciated :)